testcases/OpTestRTCdriver.py

```diff
@@ -115,14 +74,32 @@ def test_RTC_driver(self):
                 continue
         print l_list
 
+
+    ##
+    # @brief This function will cover following test steps
+    #        1. Getting host information(OS and Kernel info)
+    #        2. Loading rtc_opal module based on config option
+    #        3. Testing the rtc driver functions
+    #                Display the current time,
+    #                set the Hardware Clock to a specified time,
+    #                set the Hardware Clock from the System Time, or
+    #                set the System Time from the Hardware Clock
+    #                keep the Hardware clock in UTC or local time format
+    #                Hardware clock compare, predict and adjust functions
+    #                Hardware clock debug and test modes
+    #                Reading the Hardware clock from special file instead of default
+    #        4. After executing above each function reading the Hardware clock in b/w functions.
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def runTest(self):
+        self.test_init()
```

Comment:
Ahh.. well spotted. The test_ prefix can be magic in python unittests, so I have now changed it to rtc_init, and changed it to i2c_init in the I2C tests.